Kudu.Core/SourceControl/Git/GitExeRepository.cs

```diff
@@ -367,19 +367,28 @@ public void ClearLock()
         {
             // Delete the lock file from the .git folder
             var lockFilesPath = Path.Combine(_gitExe.WorkingDirectory, ".git");
+            // Always clear lock file under ".git/refs/heads"
+            var branchLockFiles = Path.Combine(_gitExe.WorkingDirectory, ".git", "refs", "heads");
 
             if (!Directory.Exists(lockFilesPath))
             {
                 return;
             }
 
-            var lockFiles = Directory.EnumerateFiles(lockFilesPath, "*.lock", SearchOption.AllDirectories)
+            List<string> lockFiles = Directory.EnumerateFiles(lockFilesPath, "*.lock", SearchOption.AllDirectories)
```

Comment:
yep.. i just change "var" to "List<string>", since i will do "AddRange" later, thought this will be easier to read the code when we look back the code later.